app/views/devise/sessions/_navbar.html.erb

```diff
@@ -0,0 +1,29 @@
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+
+<nav class="navbar navbar-expand-lg bg-body-tertiary bg-dark border-bottom border-body" data-bs-theme="dark">
+  <div class="container-fluid">
+    <%= link_to "Sistema de Gestion Eventos", events_main_path , class: "btn btn-dark text-light"%>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+        <li class="nav-item">
+          <% if signed_in? && current_page?(events_main_path)%>
+            <%= link_to "Iniciar", events_index_path , class: "btn btn-dark text-light"%>
+          <% end %>
+        </li>
```

Comment:
What happens when the user is not signed in or the current page is not equals to events_main_path?.
If the condition is not met do you'll leave an empty li tag?

Same for the other li tag and for the app/views/events/_navbar.html.erb file